04.ls/ls.rb

```diff
@@ -12,7 +12,7 @@ def main
 
   option.parse!(ARGV)
   filenames = options[:all] ? Dir.glob('*', File::FNM_DOTMATCH) : Dir.glob('*')
-  filenames.reverse! if options[:reverse]
+  filenames = options[:reverse] ? filenames.reverse : filenames
```

Comment:
この場合、こんなふうに書いた方が少しシンプルになります。

```suggestion
  filenames = filenames.reverse if options[:reverse]
```

ですが、変数への再代入はなるべく避けた方がいいので以下の記事を読んでみてください〜。

[Docs: プログラミング初心者は変数の再代入をなるべく避けよう \| FBC](https://bootcamp.fjord.jp/pages/avoid-re-assign)